api/v1alpha2/cluster_types.go

```diff
@@ -22,30 +22,13 @@ import (
 	capierrors "sigs.k8s.io/cluster-api/pkg/errors"
 )
 
+// +genclient
```

Comment:
Do we need genclient? I thought we were moving towards just the controller-runtime client.